DigitalLearningSolutions.Web/Views/FindYourCentre/Index.cshtml

```diff
@@ -1,9 +1,10 @@
-﻿@using DigitalLearningSolutions.Web.Helpers
```

Comment:
Ah, super minor, but can you make sure you have all these changes: (copied from Steve's MR)

```
# Configuring text editor
To get useful git diffs and make code reviewing easier, it's helpful to have the same line separator and file encoding settings on our text editors.

Line separators should be CRLF (and git auto-CRLF should be turned off).
File encoding should be UTF-8 with BOM.

In JetBrains Rider these can be set in `Settings > Editor > File Encodings` (UTF-8, with BOM) and `Settings > Editor > Code Style` (CRLF).
```

Reply:
I do, the reason the diffs are all weird here is that these files were all set to LF. They should be CRLF now 🙂